bool FEMAL=true
bool MALE=false

string first_name
string last_name
bool gender
uint8 age
string address

rosidl_tutorials_msgs/Contact[] address_book